prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -492,7 +493,12 @@ if [[ ! -z "$EXTRA_WORKFLOW" ]]; then
 fi
 
 if [[ ! -z "$ADD_EXTRA_WORKFLOW" ]]; then
-  add_W $ADD_EXTRA_WORKFLOW
+  OLD_IFS=$IFS
+  IFS=','
+  for wf in $ADD_EXTRA_WORKFLOW; do
+    add_W $wf
```

Comment:
I'd add `[[ ! -z "$wf" ]] && add_W $wf`.
your loop will not strip commas at the beginning and end of the string, i.e. something like
`ADD_EXTRA_WORKFLOW=",o2-tpc-reco-workflow"` would try to add an empty workflow.

Reply:
Ok, it was working also before, but ok with me. The loop already removes the "," since it is used as a delimiter:
```
[O2PDPSuite/latest] ~/work/O2/LHC22m_apass2/20230119 $> ADD_EXTRA_WORKFLOW=ppp,qqq
[O2PDPSuite/latest] ~/work/O2/LHC22m_apass2/20230119 $> OLD_IFS=$IFS
[O2PDPSuite/latest] ~/work/O2/LHC22m_apass2/20230119 $>   IFS=','
[O2PDPSuite/latest] ~/work/O2/LHC22m_apass2/20230119 $>   for wf in $ADD_EXTRA_WORKFLOW; do
> echo $wf
> done
ppp
qqq
```
Or am I missing your doubt?

Reply:
It will remove the `,`, but if it starts with the comma, I assume it might use the empty part before the comma as an empty string.